c/include/libsbp/v4/signing/MSG_ED25519_SIGNATURE.h

```diff
@@ -39,6 +39,22 @@ extern "C" {
  *
  *****************************************************************************/
 typedef struct {
+  /**
+   * Signature message counter. Zero indexed and incremented with each signature
+   * message. The counter will not increment if this message was in response to
+   * an on demand request. The counter will roll over after 256 messages. Upon
+   * connection, the value of the counter may not initially be zero.
+   */
+  u8 stream_counter;
+
+  /**
+   * On demand message counter. Zero indexed and incremented with each signature
+   * message sent in response to an on demand message. The counter will roll
+   * over after 256 messages. Upon connection, the value of the counter may not
+   * initially be zero.
+   */
+  u8 on_demand_counter;
+
```

Comment:
I think one `sbp_msg_ed25519_signature_t` would have either of the counter fields, maybe those should be defined as an `union`?

Reply:
I don't think the spec/codegen have support for that. A 16 byte bitfield would be kind of close. But, the requirements specify two separate fields